app/src/main/res/values-it/strings.xml

```diff
@@ -21,148 +21,148 @@
 
     <!-- Symptom -->
-    <string name="inform_symptom_hals">"Halsschmerzen"</string>
+    <string name="inform_symptom_hals">"Mal di gola"</string>
 
     <!-- Symptom -->
-    <string name="inform_symptom_kurzatmigkeit">"Kurzatmigkeit"</string>
+    <string name="inform_symptom_kurzatmigkeit">"Fiato corto"</string>
 
     <!-- Symptom -->
-    <string name="inform_symptom_fieber">"Fieber, Fiebergefühl"</string>
+    <string name="inform_symptom_fieber">"Febbre"</string>
 
     <!-- Symptom -->
-    <string name="inform_symptom_muskel">"Muskelschmerzen"</string>
+    <string name="inform_symptom_muskel">"Dolori muscolari"</string>
 
     <!-- Symptome -->
-    <string name="inform_no_symptom">"Keine der aufgeführten Symptome"</string>
+    <string name="inform_no_symptom">"Nessuno dei sintomi elencati"</string>
 
     <!-- Text in toast when downloading pdf -->
-    <string name="downloading_pdf">"PDF wird heruntergeladen"</string>
+    <string name="downloading_pdf">"Scarica il PDF"</string>
 
     <!-- Inform: Empfehlung -->
-    <string name="inform_empfehlung">"Empfehlung"</string>
+    <string name="inform_empfehlung">"Raccomandazioni"</string>
 
     <!-- Inform: Empfehlung Text -->
-    <string name="inform_empfehlung_subtext">"Eine medizinische Abklärung oder ein Labortest ist nicht angezeigt.
+    <string name="inform_empfehlung_subtext">"Un controllo medico o un test di laboratiorio al momento non è consigliato.
 
-Halten Sie sich an die Hygiene- und Verhaltensregeln."</string>
+Continua a praticare le misure di distanziamento sociale e igeniche come richiesto."</string>
 
     <!-- Inform: Button Titel -->
-    <string name="inform_fertig_button_title">"Fertig"</string>
+    <string name="inform_fertig_button_title">"Fatto"</string>
 
     <!-- CSV wird heruntergeladen -->
-    <string name="downloading_csv">"CSV wird heruntergeladen"</string>
+    <string name="downloading_csv">"Scarica il file CSV"</string>
     <string name="handshakes_heading_homescreen">" "</string>
-    <string name="handshakes_title_homescreen">"Begegnungen"</string>
+    <string name="handshakes_title_homescreen">"Incontri"</string>
     <string name="inform_heading_homescreen">" "</string>
-    <string name="inform_title_homescreen">"Verdacht auf eine Corona Erkrankung"</string>
-    <string name="inform_text_homescreen">"Informiere Personen, denen du begegnet bist, wenn du Symptome verspürst oder positiv getestet wurdest."</string>
-    <string name="inform_button_title">"Erkrankung melden"</string>
-    <string name="inform_button_title_again">"Erneut informieren"</string>
-    <string name="inform_last_meldung_text">"Letzte erfolgreiche Meldung am {date}"</string>
+    <string name="inform_title_homescreen">"Sospetto COVID-19"</string>
+    <string name="inform_text_homescreen">"Informa le persone che hai recentemente incontrato che hai i sintomi o il test al COVID-19 è risultato positivo."</string>
```

Comment:
Informa le persone che hai incontrato che hai i sintomi o che il test da COVID-19 è risultato positivo.

Reply:
(Version with "recently"): Informa le persone che hai incontrato recentemente che hai i sintomi o che il test da COVID-19 è risultato positivo.

Also, to me make more sense with this slight change (position of Codid-19 keyword):
 Informa le persone che hai incontrato recentemente che hai i sintomi da COVID-19 o che il test è risultato positivo